verification/tests/endpoints/sourcify.rs

```diff
@@ -0,0 +1,45 @@
+#[cfg(test)]
+mod tests {
```

Comment:
It is not required to put tests under a separate `#[cfg(test)]` module, as for integration tests `#[cfg(test)]` is implicitly implied.